apply_neighbourhood.json

```diff
@@ -90,7 +90,7 @@
         },
         {
             "name": "overlap",
-            "description": "Overlap of chunks along each dimension. This allows neighbourhoods to overlap. For instance a temporal dimension can add 1 month before and after a neighbourhood. In the spatial dimensions, this is often a number of pixels. The overlap specified is added before and after, so an overlap of 8 pixels will add 8 pixels on both sides of the window, so 16 in total.\nModifying overlapping data in subsequent operations should have no effect.\nFor dimensions not specified, the default is to provide no overlap.",
+            "description": "Overlap of neighbourhoods along each dimension. For instance a temporal dimension can add 1 month before and after a neighbourhood. In the spatial dimensions, this is often a number of pixels. The overlap specified is added before and after, so an overlap of 8 pixels will add 8 pixels on both sides of the window, so 16 in total.\nModifying overlapping data in subsequent operations should have no effect.\n\nOverlaps are necessary to avoid border effects, but large overlaps also increase the need for computational resources. This trade-off should be kept in mind when designing your process graph.",
```

Comment:
We don't use process graph in user communication any longer. Should just be process.